### ISSUE TITLE
[Info - Question] Optimization Process

### ISSUE THREAD

**Issue by Meywether** [entitize/Fakeddit#5]
Dear entitize,

may I ask for further information regarding you optimization process:

- Have you normalized your data? z-norm? Whole dataset or batchwise?

- Have you trained your model in an-to-end manner or have you trained the modality models seperately?

Why I am asking: I build a network architecture based on your paper but my scores are areound 5-10 % lower than mentioned in your paper!

Best regards
Meywether

**Comment by entitize** [comment]
Hi Meywether,

- We extracted features from (VGG,BERT,etc.) for specific size vectors. The whole dataset was processed. The images were resized to fit the input size vector for the image neural network. For example, VGG16 has an input size of 224x224, so we scaled the images to 224x224. For the text, we applied filtering as described in the paper such as removing punctuation, revealing words, etc. These features were then used for classification. 

- We utilized separate image and text models for feature extraction (VGG,BERT,etc.). Once all features have been extracted we trained a separate model to combine these extracted features and classify our input.

Does this answer your questions?

**Comment by Meywether** [comment]
Thanks a lot for you explanation, and yes it answers my questions :)
Thanks entitize!

**Comment by cococold** [comment]
> Hi Meywether,
> 
> * We extracted features from (VGG,BERT,etc.) for specific size vectors. The whole dataset was processed. The images were resized to fit the input size vector for the image neural network. For example, VGG16 has an input size of 224x224, so we scaled the images to 224x224. For the text, we applied filtering as described in the paper such as removing punctuation, revealing words, etc. These features were then used for classification.
> * We utilized separate image and text models for feature extraction (VGG,BERT,etc.). Once all features have been extracted we trained a separate model to combine these extracted features and classify our input.
> 
> Does this answer your questions?

Dear entitize,
May I ask for further details about the optimization process:
    1. As you mentioned above, I thought the Bert model or ResNet model is not fine-tuned in the optimization process. So, did you have fine-tuned them before the feature extraction, or just utilize the open-source pre-trained models for the feature extraction?
    2. There are three different classification tasks, did you complete them with a single model in a multitasking manner, or you trained a model for each task?

Best regards

**Comment by Frankey419** [comment]
> extraction

And, could you provide more information about the ResNet+Bert_large model structure? such as the number of units in the hidden layer, and the learning rate. Sharing the codes are better.

Thanks
